test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-SecureString.Tests.ps1

```diff
@@ -1,8 +1,9 @@
-Describe "ConvertTo-SecureString" -Tags "CI" {
+	Describe "ConvertTo--SecureString" -Tags "CI" {
```

Comment:
Seems to have an extra - between ConvertTo and SecureString

Reply:
I have to rename it to something other than convertto-securestring.